interceptor.go

```diff
@@ -302,12 +308,12 @@ func (i *Interceptor) WrapStreamingHandler(next connect.StreamingHandlerFunc) co
 
 func protocolToSemConv(protocol string) string {
 	switch protocol {
-	case "grpcweb":
-		return "grpc_web"
-	case "grpc":
-		return "grpc"
-	case "connect":
-		return "buf_connect"
+	case grpcwebString:
+		return grpcwebProtocol
+	case grpcProtocol:
+		return grpcProtocol
+	case connectString:
+		return connectProtocol
```

Comment:
🤷🏽‍♂️  No objections to this, but I've never quite understood why the extra indirection is worth it if we're only referencing the string in one place.

Reply:
tbh I think it's probably just because it looks prettier:
```

	case grpcProtocol:
		return grpcProtocol
	case connectString:
		return connectProtocol
```

vs 

```

	case grpcProtocol:
		return grpcProtocol
	case "connect":
		return connectProtocol
```